aws/cloudtrail/main.tf

```diff
@@ -11,6 +11,13 @@ resource "aws_cloudtrail" "mod" {
   tags                          = var.tags
 }
 
+resource "aws_s3_bucket" "logs" {
+  bucket = "${var.name}-cloudtrail-logs"
```

Comment:
To avoid potential name collisions, maybe consider using [`bucket_prefix`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#bucket_prefix) instead of the exact name?